### DISCUSSION TITLE
Adding privacy statement to https://www.scriptkit.com/ website

### DISCUSSION THREAD

**Discussion by Rafi993** [johnlindquist/kit#296] [General]
Thank you for this awesome product. It would be nice if there is a small privacy section at the bottom of the website https://www.scriptkit.com/ would really help. Even a one-liner would help people who might be on the fence about installing the product about its privacy concerns.

**Comment by johnlindquist** [comment]
Like a "Kit.app collects zero user information"? Is that the concern?
